packages/lazy-images/.gitattributes

```diff
@@ -0,0 +1,7 @@
+# Files not needed to be distributed in the package.
+.gitattributes export-ignore
+phpunit.xml    export-ignore
+tests/         export-ignore
+composer.lock  export-ignore
+vendor/        export-ignore
+wordpress/    export-ignore
```

Comment:
You may not need this anymore?

Reply:
I actually mis-undertood the meaning of this. I understand now. Thanks for pointing it out